### PR TITLE
Renamed ExeName to Command, added Description in start info

### PR DESCRIPTION
**Pull request by dhruv-vora**
**Description:**
- Renamed ExeName to Command
- Added Description to store the full name of the collector

### CODE REVIEW

**Comment by dhruv-vora** [comment]
@bogdandrutu @Aneurysm9 @anuraaga @alolita

**Comment by rakyll** [comment]
Can you rebase this PR? It has commits from other PRs.

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,8 +10,10 @@
 - Rename `marshall` to `marshal` in all the occurrences (#2977)
 - Remove `componenterror.ErrAlreadyStarted` and `componenterror.ErrAlreadyStopped`, components should not protect against this, Service will start/stop once.
 - Rename `ApplicationStartInfo` to `BuildInfo`
+- Rename `ApplicationStartInfo.ExeName` to `BuildInfo.Command`
 
 ## 💡 Enhancements 💡
+ - Add `Description` to `BuildInfo`
```
Comment (bogdandrutu) [inline CHANGELOG.md:16]: This is actually a rename of the `ApplicationStartInfo.LongName`

Comment (dhruv-vora) [reply CHANGELOG.md]: I have updated this

**Review by bogdandrutu — APPROVED** [review]

**Pull request merged**